packages/0x.js/test/0x.js_test.ts

```diff
@@ -31,7 +49,7 @@ describe('ZeroEx library', () => {
             expect((zeroEx.exchange as any)._exchangeContractIfExists).to.not.be.undefined();
             expect((zeroEx.tokenRegistry as any)._tokenRegistryContractIfExists).to.not.be.undefined();
 
-            const newProvider = web3Factory.getRpcProvider();
+            const newProvider = web3.currentProvider;
```

Comment:
Please try to minimize the usage of web3 when possible. Just export the provider from that module above. Don't use the web3 object to just carry the provider.